### PR TITLE
New package: n2n-3.0

### PR DESCRIPTION
**Pull request by strec007**
#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (amd64)

### CODE REVIEW

File: srcpkgs/n2n/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version=3.0
+revision=1
+build_style=gnu-configure
+configure_args="SBINDIR=/usr/bin"
+hostmakedepends="autoconf automake pkg-config"
+short_desc="Light VPN software"
+maintainer="Petr Cizmar <xxb0322@xpc.ovh>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/ntop/n2n"
+changelog="https://raw.githubusercontent.com/ntop/n2n/${version}-stable/CHANGELOG.md"
```
Comment (tranzystorek-io) [inline srcpkgs/n2n/template:13]:
```suggestion
changelog="https://raw.githubusercontent.com/ntop/n2n/dev/CHANGELOG.md"
```

File: srcpkgs/n2n/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'n2n'
+
```
Comment (abenson) [inline srcpkgs/n2n/template:2]:
```suggestion
```

Comment (strec007) [reply srcpkgs/n2n/template:2]: That would break the xlint check.

Comment (abenson) [reply srcpkgs/n2n/template:2]: No it wouldn't.

**Comment by strec007** [comment]
OK, now seriously, is any maintainer interested in merging n2n? If not, I'm not gonna bother fixing the details.

**Pull request closed without merging**